structcol/montecarlo.py

```diff
@@ -167,15 +203,28 @@ def scatter(self, sintheta, costheta, sinphi, cosphi):
         # idea to hard-code constants like this. What determines epsilon? What
         # happens if you change the value? See if you can rewrite the code so
         # that the tolerance factor either isn't needed (consider using np.isclose()).
-        # also explain what you are doing here.
-        eps = 1e-4
 
-        #TODO: please explain and comment what is going on here.  The code is very opaque.
+        # RESPONSE: I used the epsilon factor because [1] recommends using
```

Comment:
you need to write this as a comment explaining what you are doing in np.isclose(), not as a response to my comment.